galasa-parent/dev.galasa.framework/src/main/java/dev/galasa/framework/spi/IManager.java

```diff
@@ -85,6 +86,9 @@
      */
     void initialise(@NotNull IFramework framework, @NotNull List<IManager> allManagers,
             @NotNull List<IManager> activeManagers, @NotNull Class<?> testClass) throws ManagerException;
+
+    void registerStatements(@NotNull IFramework framework, @NotNull List<IManager> allManagers,
```

Comment:
think this needs to be in initialise, ie pass testClass or gherkinTest to initialise and then the manager can parse then.
Or, pass an object that has TestClass or GherkinObject, that way we future proof if we add another language